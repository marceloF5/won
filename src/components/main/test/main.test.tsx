import { render, screen } from '@testing-library/react'

import Main from '../main'

describe('<Main />', () => {
    it('should render Main component', () => {
        const { container } = render(<Main />)

        expect(container.firstChild).toMatchSnapshot()

        expect(
            screen.getByRole('heading', { name: /Advanced React Course/i })
        ).toBeInTheDocument()
    })
})
