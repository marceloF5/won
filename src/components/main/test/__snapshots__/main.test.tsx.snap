// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render Main component 1`] = `
<main>
  <h1>
    Advanced React Course
  </h1>
</main>
`
